Heimdall/Heimdall.swift

```diff
@@ -882,18 +894,21 @@ private extension NSData {
                 offset += OID.count
                 
                 // Type
-                if bytes[offset++] != 0x03 {
+                if bytes[offset] != 0x03 {
```

Comment:
Should the contents of this if also increment the `offset` key? If it is true, then currently the offset value is not incremented, which did happen previously (right?).